crates/networking/p2p/src/network.rs

```diff
@@ -642,4 +913,84 @@ mod tests {
             }
         });
     }
+
+    #[test]
+    fn test_peer_table_lifecycle() {
+        initialize_network_spec();
+
+        let tokio_runtime = Runtime::new().unwrap();
+        let mut network1 = tokio_runtime
+            .block_on(create_network(
+                "127.0.0.1".parse().unwrap(),
+                9300,
+                9301,
+                vec![],
+                true,
+                vec![],
+            ))
+            .unwrap();
+
+        let mut network2 = tokio_runtime
+            .block_on(create_network(
+                "127.0.0.1".parse().unwrap(),
+                9302,
+                9303,
+                vec![],
+                true,
+                vec![],
+            ))
+            .unwrap();
+
+        let address: Multiaddr = "/ip4/127.0.0.1/tcp/9300".parse().unwrap();
+        network2.swarm.dial(address).unwrap();
+
+        let peer_id_network1 = network1.peer_id();
```

Comment:
```suggestion
        let address: Multiaddr = "/ip4/127.0.0.1/tcp/9300".parse().unwrap();
        network_2.swarm.dial(address).unwrap();

        let peer_id_network_1 = network_1.peer_id();
```

we sure we have underscores between these to be consistent, check your PR for other cases this happens